test/modules/aws-acm-import-integration.ts

```diff
@@ -148,7 +102,7 @@ describe('AwsAcmImport Integration Testing', () => {
     .then(...finish(done)));
 });
 
-describe('AwsAcmImport install/uninstall', () => {
+describe('AwsAcm Import install/uninstall', () => {
```

Comment:
this last describe would not be necessary since it is not a separate module anymore